### ISSUE TITLE
Dynamic import has conflicts with rule comma-dangle

### ISSUE THREAD

**Issue by lgraziani2712** [babel/babel-eslint#507]
Hello!

### Description of the bug

I'm having this issue when the import's param  is long enough to reach the max-len rule, so I break it into two lines. But then Eslint ask me to add a trailling comma, when dynamic imports don't allow them.

### Example

- Before: max-len warning.
```jsx
import(/* webpackChunkName: "long-enough" */ `this/is/long/enough/to/${require}/a/line/break`);
```

- After: comma-dangle warning.

```jsx
import(
  /* webpackChunkName: "long-enough" */ `this/is/long/enough/to/${require}/a/line/break`
);
```

Thank you!

**Comment by lgraziani2712** [comment]
I don't know when, but it is resolved 🎉 

Thank you!

**Comment by lgraziani2712** [comment]
Nevermind. It still generates the comma. 😢

**Comment by jcperez-ch** [comment]
Apparently after almost a year, this is still an issue.  Not sure we there would be a workaround but this issue is related to https://github.com/babel/babel/issues/8957 . I think both issues are valid since it affects users.

**Comment by nunoleong** [comment]
any news on this ? it still seems to be an issue

**Comment by kaicataldo** [comment]
Thank you for the issue. Now that [`@babel/eslint-parser` has been released](https://babeljs.io/blog/2020/07/13/the-state-of-babel-eslint), we are making this repository read-only. If this is a change you would still like to advocate for, please reopen this in the [babel/babel monorepo](https://github.com/babel/babel).
